package.json

```diff
@@ -19,7 +19,7 @@
     "lib": "lib"
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=18"
```

Comment:
Could we just include on the release notes, we definitely have the all clear for this yeah?

Reply:
Yes I will add it on the publish PR